lib/page/detail_page.dart

```diff
@@ -97,23 +199,31 @@ class DetailPage extends StatelessWidget {
 List<Text> getLocationList(Location? location) {
   List<Text> filteredLocationTextList = [];
   if (location?.address != null) {
-    filteredLocationTextList.add(Text(
-      location!.address!,
-      style: TextStyle(fontSize: 20, fontWeight: FontWeight.bold),
-    ));
+    filteredLocationTextList.add(
+      Text(
+        location!.address!,
```

Comment:
since you have `if (location?.address != null)` check on line 201, I think you won't need `?` here anymore. Therefore you can try to put just `location.address`

Reply:
error

Reply:
done